api/src/main/java/com/stormpath/sdk/group/Group.java

```
@@ -166,6 +166,47 @@
     GroupMembership addAccount(Account account);
 
     /**
+     * Assigns this Group to the specified Account represented by its (case insensitive) {@code username}, {@code email} or {@code href}
+     * <p/>
+     * <b>Immediate Execution:</b> Unlike other Group methods, you do <em>not</em> need to call {@link #save()} afterwards.
+     * This method will persist the changes in the backend immediately.
+     *
+     * @param hrefOrEmailOrUsername the href, email or username of the Account to associate.
+     * @return the new GroupMembership resource created reflecting the group-to-account association.
+     *
+     * @since 1.0.RC5
+     */
+    GroupMembership addAccount(String hrefOrEmailOrUsername);
+
+    /**
+     * Removes this Group's association with the specified Account.
+     *
+     * <p><b>Immediate Execution:</b> Unlike other Group methods, you do <em>not</em> need to call {@link #save()}
+     * afterwards. This method will persist the changes in the backend immediately.</p>
+     *
+     * @param account the {@code Account} object to disassociate.
+     * @return the Group object for method chaining
+     * @throws IllegalStateException if the specified Account does not belong to this Group.
+     *
+     * @since 1.0.RC5
+     */
+    Group removeAccount(Account account);
```

Should throw an exception if the account is not already in the group